code-challenges.test.js

```diff
@@ -63,5 +101,16 @@ const vowelTester3 = "challenges"
 // Expected output: 2
 
 // b) Create the function that makes the test pass.
-
+const firstVowel = (str) => {
+  let vowels = ["a", "e", "i", "o", "u"]
+  for(i = 0; i < str.length; ++i) {
+    if(vowels.includes(str[i])) {
+      return i
+    }
+  };
+}
 // Pseudo code:
+// create function with 1 input 
+// create variable array that holds all vowels 
+// forEach loop that stops when includes method === true
```

Comment:
You said forEach here when the code is using a for loop, make sure your code reflects your pseudo

Reply:
gotchya. I wrote the pseudo then did the code and never went back. I'll be more careful next time.